src/pages/index.js

```diff
@@ -0,0 +1,331 @@
+//     alt: "Val Thorens",
+//   },
+//   {
+//     name: "Restaurant terrace",
+//     src: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//     alt: "Restaurant terrace",
+//   },
+
+//   {
+//     name: "An outdoor cafe",
+//     src: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//     alt: "An outdoor cafe",
+//   },
+
+//   {
+//     name: "A very long bridge, over the forest and through the trees",
+//     src: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//     alt: "A very long bridge, over the forest and through the trees",
+//   },
+
+//   {
+//     name: "Tunnel with morning light",
+//     src: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//     alt: "Tunnel with morning light",
+//   },
+
+//   {
+//     name: "Mountain house",
+//     src: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//     alt: "Mountain house",
+//   },
+// ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7d7c7032-a925-465f-95d5-699a1df9381c",
+    about: "Placeholder description",
+    avatar:
+      "https://practicum-content.s3.amazonaws.com/resources/default-avatar_1704458546.png",
+    name: "Placeholder name",
+    id: "",
+    "Content-Type": "application/json",
+  },
+});
+
+// destructure second item in callback of .then()
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((card) => {
+      const cardElement = getCardElement(card);
+      cardsList.append(cardElement);
+    });
+    profileImage.src = userInfo.avatar;
+    profileNameElement.textContent = userInfo.name;
+    profileDescriptionElement.textContent = userInfo.about;
+  })
+  .catch(console.error);
+
+// Profile Elements
+const profileEditButton = document.querySelector(".profile__edit-button");
+const profileAddButton = document.querySelector(".profile__add-button");
+const profileNameElement = document.querySelector(".profile__name");
+const profileDescriptionElement = document.querySelector(
+  ".profile__description"
+);
+
+const previewModal = document.querySelector("#preview-modal");
+const preModalImage = previewModal.querySelector(".modal__image");
+const preModalCaptionEl = previewModal.querySelector(".modal__caption");
+
+// Avatar Elements
+const avatarModal = document.querySelector("#edit-avatar-modal");
+const avatarEditButton = document.querySelector(".profile__avatar-btn");
+const avatarInput = avatarModal.querySelector("#profile-avatar-input");
+const avatarForm = avatarModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms